.github/workflows/build-pr-status-check.yml

```
@@ -31,7 +31,7 @@ jobs:
       run: |
         $pullRequestNumber = "${{ github.ref }}".Replace("refs/pull/", "").Replace("/merge", "");
         Write-Host "Pull Request Number: $pullRequestNumber";
-        Write-Host "::set-output name=pr-number::$pullRequestNumber";
+        "pr-number=$pullRequestNumber" >> $GITHUB_OUTPUT;
```

This step output is incorrect.
This needs to be done for _**ALL**_ of the workflows.
Update this step output to the following:

```suggestion
        "pr-number=$pullRequestNumber" >> $env:GITHUB_OUTPUT;
```

---

This has not been completed for all of the workflows.  Search for the `$GITHUB_OUTPUT` output file env variable and convert them to what is shown in the comment.